dc/s2n-quic-dc/src/path/secret/map/state.rs

```diff
@@ -24,6 +24,119 @@ use std::{
 #[cfg(test)]
 mod tests;
 
+#[derive(Default)]
```

Comment:
Do we want to set a large-ish default capacity so we don't have to realloc a bunch as entries ramp up at the beginning?

Reply:
Fixed, we now reserve 2x the max capacity upfront. I think that ensures no more allocations, but at most one extra (if we're off by ~one somewhere, though our current defaults don't actually make that "one" since the table is a power of two -- 500k * 2 = 1MM but I'd expect it to actually be 1,048,576).